package de.thm.arsnova.service.exceptions;

public class UserAlreadyExistsException extends RuntimeException {
}
